src/transaction/types.ts

```diff
@@ -117,11 +125,13 @@ export type _UnsignedTxParsed = {
   inputs: _Input[],
   outputs: _Output[],
   fee: Lovelace,
-  ttl: number,
+  ttl?: number,
```

Comment:
this should be a BigInt as ttl is uint64 by spec

Reply:
Fixed